mcs/class/System.XML/ReferenceSources/LocalAppContextSwitches.cs

```diff
@@ -5,5 +5,7 @@ static class LocalAppContextSwitches {
 		public static readonly bool DontThrowOnInvalidSurrogatePairs = false;
 		public static readonly bool IgnoreKindInUtcTimeSerialization = false;
 		public static readonly bool EnableTimeSpanSerialization = false;
+		public static readonly bool LimitXPathComplexity = true;
+		public static readonly bool AllowDefaultResolver = true;
```

Comment:
are we sure these are the correct default values for us? aren't AppContext switches usually false and you need to use `true` to opt into the old/bad behavior?

Reply:
yeah, they should be `false` by default, thanks for noticing!